readme.md

```diff
@@ -367,7 +367,7 @@ Default:
 - limit: `2`
 - calculateDelay: `(attemptCount, retryOptions, error, computedValue) => computedValue`
 - methods: `GET` `PUT` `HEAD` `DELETE` `OPTIONS` `TRACE`
-- statusCodes: [`408`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408) [`413`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413) [`429`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) [`500`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) [`502`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502) [`503`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) [`504`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504)
+- statusCodes: [`408`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/408) [`413`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/413) [`429`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) [`500`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/500) [`502`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/502) [`503`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503) [`504`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/504) [`520`](https://support.cloudflare.com/hc/en-us/articles/115003011431#520error) [`521`](https://support.cloudflare.com/hc/en-us/articles/115003011431#521error) [`522`](https://support.cloudflare.com/hc/en-us/articles/115003011431#522error) [`523`](https://support.cloudflare.com/hc/en-us/articles/115003011431#523error) [`524`](https://support.cloudflare.com/hc/en-us/articles/115003011431#524error) [`527`](https://support.cloudflare.com/hc/en-us/articles/115003011431#527error)
```

Comment:
Only `522` and `524` should be retry-able.

Reply:
Other `5XX` codes are not.

Reply:
Just so I understand why, is your reasoning that the other ones are unlikely to be intermittent?

Reply:
I'm glad you've asked :smiley: Got retries on timeouts by default, so `522` and `524` are OK.

On the other hand, `520` is caused by a server error which is not likely to disappear:

![image](https://user-images.githubusercontent.com/36894700/70850338-732e7a00-1e89-11ea-9285-80e3eeeef335.png)

`521` is `web server is down`, it makes no sense to retry.

`523` is `origin is unreachable` - CloudFlare cannot find a route to the server, it makes no sense to retry either.

`527` is `Railgun Listener to origin error` and it has these common causes:

![image](https://user-images.githubusercontent.com/36894700/70850408-26976e80-1e8a-11ea-9606-fad735dea42d.png)

It isn't safe to retry on `527` as we don't know if the request timed out or not.

Reply:
Thanks for the explanation. I made the change.

Reply:
We retry on `ECONNREFUSED` and `ECONNRESET` though, so `521` is appriopriate here: https://community.cloudflare.com/t/community-tip-fixing-error-521-web-server-is-down/42461

Let me finish this for you :)